src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int buySum = 0;
+    private int supplySum = 0;
+    private final String supplyOperation = "supply";
+    private final String buyOperation = "buy";
+    private final String result = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] tempString = readFile(fromFileName).toString().split("\\W+");
+        calculateSum(tempString);
+        checkWriteFile(toFileName);
+        writeFile(toFileName);
+    }
+
+    public StringBuilder readFile(String fromFile) {
```

Comment:
all methods except getStatistic should be private

Reply:
OK